lib/taurus/core/util/log.py

```diff
@@ -793,6 +793,15 @@ def _format_stack(self, stack_func=inspect.stack):
                         out += '<could not find suitable string representation>'
         return out
 
+    def loggingtofile(self, msg, logging_path, *args, **kw):
+
+        file_name = logging_path + "/spock_session.log"
+
+        logging_file = open(file_name, 'a')
+        logging_file.write('\n')
+        logging_file.write(msg)
+        logging_file.close()
+
```

Comment:
Hi @cpascual, thanks, it was protected in the sardana part, informing about the reason of the
 error, but I put it now here and I took out the reason of the error (since now  could be another
 one) in the sardana part.